File: Objective-C Version/DynamicActionBlockDemo/DynamicActionBlockDemo/DynamicActionBlockDemo/ViewController.m
//
//  ViewController.m
//  DynamicActionBlockDemo
//
//  Created by Kitten Yang on 9/27/15.
//  Copyright (c) 2015 Kitten Yang. All rights reserved.
//

#import "ViewController.h"
#import "DynamicView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    DynamicView *dynamicView = [[DynamicView alloc]initWithFrame:self.view.bounds];
    [self.view addSubview:dynamicView];
}



- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Swift Version/DynamicActionBlockDemo-Swift/DynamicActionBlockDemo-Swift/ViewController.swift
//
//  ViewController.swift
//  DynamicActionBlockDemo-Swift
//
//  Created by Kitten Yang on 1/2/16.
//  Copyright © 2016 Kitten Yang. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let dynamicView = DynamicView(frame: view.bounds)
        view.addSubview(dynamicView)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
    }


}

